packages/experiments/src/components/Separator/Separator.types.ts

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { SeparatorBase } from './Separator.base';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+import { IStyle, ITheme } from '../../Styling';
+
+export interface ISeparator {}
+
+export interface ISeparatorProps extends React.Props<SeparatorBase> {
+  /**
+   * Optional callback to access the ISeparator interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: IRefObject<ISeparator>;
+
+  /**
+   * Theme (provided through customization.)
+   */
+  theme?: ITheme;
+
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  styles?: IStyleFunctionOrObject<ISeparatorStyleProps, ISeparatorStyles>;
+
+  /**
+   * Optional text to display in the separator.
+   */
+  text?: string;
+
+  /**
+   * Whether the element is a vertical separator.
+   */
+  vertical?: boolean;
+
+  /**
+   * Where the text should be aligned in the separator.
+   */
+  alignText?: 'start' | 'center' | 'end';
+
+  /**
+   * Optional class for separator.
+   */
+  className?: string;
+}
+
+export interface ISeparatorStyleProps {
```

Comment:
You may consider using `Pick<T, K extends keyof T>` here where `T` is `ISeparatorProps` and the keys are common fields if it helps you.

I just remember it from our Styling Wiki: https://github.com/OfficeDev/office-ui-fabric-react/wiki/Component-Styling#icomponentstyleprops